selfdrive/controls/lib/events.py

```diff
@@ -945,4 +945,10 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.NO_ENTRY: NoEntryAlert("LKAS Disabled"),
   },
 
+  EventName.vehicleSensorsInvalid: {
+    ET.IMMEDIATE_DISABLE: ImmediateDisableAlert("Vehicle Sensors Invalid"),
+    ET.PERMANENT: NormalPermanentAlert("Vehicle Sensors Calibrating", "Drive to Calibrate"),
```

Comment:
It recovers in ~30 seconds in a few routes I looked at, but it did not recover when I tested it today as I kept the car at a standstill due to flat tire. So it likely requires the vehicle to be driven.